### PR TITLE
Unit Testing plugin fixes

### PR DESCRIPTION
**Pull request by LianaHus**
fix https://github.com/ethereum/remix-project/issues/1255

### CODE REVIEW

File: apps/remix-ide/src/app/tabs/test-tab.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { ViewPlugin } from '@remixproject/engine-web'
 import { canUseWorker, urlFromVersion } from '../compiler/compiler-utils'
+import * as helper from '../../lib/helper'
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/tabs/test-tab.js:3]: You can import only methods to be used

**Comment by Aniket-Engg** [comment]
If I type a non-existing dir and press `/`, console shows error:
```
Uncaught (in promise) Error: Error: No such file or directory 
    at FileManager._callee12$ (main.js:35815)
    at tryCatch (polyfills.js:14612)
    at Generator.invoke [as _invoke] (polyfills.js:14838)
    at Generator.prototype.<computed> [as throw] (polyfills.js:14664)
    at asyncGeneratorStep (main.js:34960)
    at _throw (main.js:34962)
```

**Comment by Aniket-Engg** [comment]
I see another issue not sure if it is because of changes in this PR . If I have folder with prefix of some existing folder like `con` for `contracts` and `con` and `contracts` both are expanded in file explorer. Now if I collapse `con` directory files, `contracts` is automatically collapsed. It is not happening while expanding.

**Review by Aniket-Engg — COMMENTED** [review]
If I input `to//t`, click on `Create` and then on `generate`.
switch to the file explorer and a folder with a blank folder has been created.

**Comment by LianaHus** [comment]
> If I input `to//t`, click on `Create` and then on `generate`.
> switch to the file explorer and a folder with a blank folder has been created.

hmm I'm pretty sure this was fixed. thanks

File: apps/remix-ide/src/app/tabs/test-tab.js [hunk 2]
```diff
@@ -52,18 +54,26 @@ module.exports = class TestTab extends ViewPlugin {
...
+          if (error) return tooltip(error)
+          this.data.allTests = tests
+          this.data.selectedTests = [...this.data.allTests]
+          this.updateTestFileList(tests)
+          if (!this.testsOutput) return // eslint-disable-line
+        })
+      } catch (e) {
+        console.log(e)
+      }
       this.data.allTests.push(file)
```
Comment (yann300) [inline apps/remix-ide/src/app/tabs/test-tab.js:69]: do you need to have `this.data.allTests.push(file)` and `this.data.selectedTests.push(file)` they seem to be updated in the callback above.

**Comment by yann300** [comment]
one of the initial issue is not fixed: 
```
 - input `contracts/` will create the path default_workspace/contracts//newFile_test.sol. which is wrong.
 - switch to the file explorer and a folder with a blank folder has been created. reload remix and the folder name seems to be displayed correctly.
```

**Comment by Aniket-Engg** [comment]
Screenshot 2021-06-09 at 11 11 41 AM

**Review by Aniket-Engg — COMMENTED** [review]
Old issues seems gone but I see an issue in loading test files from selected directory. Have a look to above image.

**Review by Aniket-Engg — APPROVED** [review]
Loading of test files with updated test directory works best after clicking `Enter`

**Pull request merged**